predicators/third_party/ikfast/panda_arm/compile.py

```diff
@@ -0,0 +1 @@
+../compile.py
```

Comment:
For reference this is a symlink

Reply:
this is a clever way to deal with relative import headaches!